Include/internal/pycore_abstract.h

```diff
@@ -8,6 +8,18 @@ extern "C" {
 #  error "this header requires Py_BUILD_CORE define"
 #endif
 
+/// Return 1 if the specified index is outside the range [0, limit)
```

Comment:
I think that this comment is wrong. It returns non-zero if the index is *inside* the specific range, no?

Reply:
Good catch, I forgot to update after changing from `invalid_index` to `valid_index`